ct_core/CMakeLists.txt

```diff
@@ -63,12 +63,12 @@ else()
     message(STATUS "COMPILING WITHOUT QWT")
 endif()
 
-find_package(PythonLibs 2.7 QUIET)
-if (PYTHONLIBS_FOUND)
-   message(STATUS "Found python 2.7")
+find_package(Python COMPONENTS Development NumPy QUIET)
```

Comment:
A problem here might be that this command, as far as I am informed, is only supported for cmake version 3.12 or greater. The default cmake version on Ubuntu 16.04 LTS is 3.5. Since we have quite a few users which are on Ubuntu 16, we need a solution to deal with that problem.
- either provide a script which automatically installs a newer cmake version on user PC's
- find a solution that works with older cmake versions?

Reply:
One more thought.... you mentioned that the feature might break code running with python 2.7.
In that respect, would it make sense to include a check here, asserting that we have python3 in place, and only set the compile flag in case the python version is sufficient?

Reply:
Since `FindPythonLibs` is a deprecated feature, I think it would be best to have a script that installs a newer cmake version for Ubuntu 16 & 18. Although getting it to work with an older cmake that is available in the Ubuntu repos is a cleaner solution. Let me see what I can do.

This feature shouldn't break code running with python 2.7. It just means that control toolbox will not work with python 2.7. However, adding the check is a smart thing to do.

Reply:
I added a check for python 3.

I ended up going with providing a script that installs a newer version of cmake. (3.14.7) I bumped up the minimum cmake to reflect this. The reason I ended up going with a newer cmake is because the older feature will be deprecated and we will have to change it anyway in the future. As for 3.14.7 it was the latest release with had support for find numpy include dirs.

Reply:
very cool. 
Would you mind adding the line `cmake_minimum_required (VERSION 3.14.7)` to the other CMakeLists.txt in ct_optcon, ct_rbd and ct_models? Thanks a lot in advance!

Reply:
oh shucks forgot about that, on it.

Reply:
Thanks, now also works for me!